src/templates/default/reviews/review_stages.html

```diff
@@ -0,0 +1,114 @@
+		{% for rs in review_stages_list %}
+		<button class="btn btn-natural-noborder hotkey" value="{{ rs }}">{{ rs }}</button>
+		{% endfor %}
+	</div>
+	<!--Review Change Setting-->
+	<div class="col-md-9">
+		<form action="{% url 'review_stages' %}" method="post">
+			{% csrf_token %}
+			<div class="panel panel-default">
+				<div class="panel-heading">{% trans 'Review Stages' %}
+					<div class="material-switch pull-right">
+						<a href="https://github.com/pycontw/pycon.tw/blob/master/src/reviews/README.md">See Proposal Review System
+							Documentation</a>
+					</div>
+				</div>
+				<ul class="list-group">
+					<li class="panel-title list-group-item">
+						{{ ".proposals.creatable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.creatable" type='hidden' value="false" />
+							<input id="proposals.creatable" name="proposals.creatable" type="checkbox" value="true"/>
+							<label for="proposals.creatable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.editable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.editable" type='hidden' value="false" />
+							<input id="proposals.editable" name="proposals.editable" type="checkbox" value="true"/>
+							<label for="proposals.editable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.withdrawable"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="proposals.withdrawable" type='hidden' value="false" />
+							<input id="proposals.withdrawable" name="proposals.withdrawable" type="checkbox" value="true"/>
+							<label for="proposals.withdrawable" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.visible.to.submitters"|configuration_switch }}
+						<div class="material-switch pull-right">
+							<input name="reviews.visible.to.submitters" type='hidden' value="false" />
+							<input id="reviews.visible.to.submitters" name="reviews.visible.to.submitters"
+								type="checkbox" value="true"/>
+							<label for="reviews.visible.to.submitters" class="label-info"></label>
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".reviews.stage"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input id="reviews.stage" name="reviews.stage" type="number" min="0" max="2"
+								placeholder="Int" required="required">
+						</div>
+					</li>
+					<li class="panel-title list-group-item">
+						{{ ".proposals.disable.after"|configuration_switch }}
+						<div class="input-customized-size pull-right">
+							<input name="proposals.disable.after" type="datetime-local" step="1"
+								placeholder="yyyy-mm-dd hh:mm:ss" required="required">
```

Comment:
I am having doubts about making this field "required" because I think we should be able to disable it by not providing any value?

And another thing is that the backend needs the format exactly as `YYYY-mm-ddThh:mm:ss`, you need to put the T in there, but the placeholder hints without the T. I personally think that `YYYY-mm-dd hh:mm:ss` should be more user-friendly, and you can consider accepting values without `ss` (treat missing `ss` as 0).

You can take a look at how django implements this here:
https://github.com/django/django/blob/master/django/forms/fields.py#L366

They take multiple possible formats and try one another, described in here https://docs.djangoproject.com/en/3.0/ref/settings/#datetime-input-formats

I think this is a little bit too much, but if you have time you can try this. The bottom line is at least make it more user friendly and the frontend hint needs to meet the backend validation criteria.

Reply:
We want to make it disable if no value is provided. Is it convenient for users to modify only a single value?

Reply:
But on the other hand, what does it mean that we should be able to disable it by not providing any value ? Is it for users to change a value individually?

Reply:
What I meant is that, `proposals.disabled.after` should take effect if this key / value is set. If no value is set for this key, like, an empty value, then it will never be disabled after any time.